### DISCUSSION TITLE
[Question][Hw4] About coverage set

### DISCUSSION THREAD

**Discussion by SangHyeon-Lee** [prosyslab-classroom/cs348-information-security#572] [Q&A]
Hello, 
I have a question regarding set of coverage.

As far as I know, it seems I have to update cumulated set of coverage by update_seeds.
And professor and TAs said that the evaluation will be held by collecting all .cov files.
What is the role of this coverage set?
Is it correct to update cumulated set of coverage?

Thank you in advance!

**Comment by hieudz** [comment]
Im not the TA, but I believe that coverage shows how many instructions your inputs have passed. And how you update the coverage set in your code doesn't matter, TA eill only grade the output coverage files. But I also accumulate the set of coverage in my code to improve the mutation

**Comment by RiceBiscuits** [comment]
You don't need to accumulate .cov files yourself. TA will do that during the evaluation.
We can measure opportunity of finding bugs by coverage.

**Comment by SangHyeon-Lee** [comment]
Thank you for all answers!
